compilers/concrete-compiler/compiler/lib/Bindings/Python/concrete/compiler/client_support.py

```diff
@@ -175,30 +175,34 @@ def decrypt_result(
             raise TypeError(
                 f"public_result must be of type PublicResult, not {type(public_result)}"
             )
-        lambda_arg = LambdaArgument.wrap(
-            _ClientSupport.decrypt_result(
-                client_parameters.cpp(), keyset.cpp(), public_result.cpp()
-            )
+        results = _ClientSupport.decrypt_result(
+            client_parameters.cpp(), keyset.cpp(), public_result.cpp()
         )
 
-        output_signs = client_parameters.output_signs()
-        assert len(output_signs) == 1
+        def process_result(result):
+            lambda_arg = LambdaArgument.wrap(result)
+            is_signed = lambda_arg.is_signed()
+            if lambda_arg.is_scalar():
+                return (
+                    lambda_arg.get_signed_scalar()
+                    if is_signed
+                    else lambda_arg.get_scalar()
+                )
 
-        is_signed = lambda_arg.is_signed()
-        if lambda_arg.is_scalar():
-            return (
-                lambda_arg.get_signed_scalar() if is_signed else lambda_arg.get_scalar()
-            )
+            if lambda_arg.is_tensor():
```

Comment:
same, and elif is more adapted, i.e. exlusive cases (if the linter is not too stupid to reject it)

Reply:
I believe the standard style is `if: return` pattern